src/pynmeagps/nmeamessage.py

```diff
@@ -238,7 +238,10 @@ def _get_dict(self, **kwargs) -> dict:
             key = self.msgID
             if key in nmt.PROP_MSGIDS:  # proprietary, first element is msgId
                 if "payload" in kwargs:
-                    key += self._payload[0]
+                    if key == "ASHR" and self._payload[0][1].isdigit():

```

Comment:
I take it PASHR msgId can never begin with a digit?

Reply:
yes, according to https://receiverhelp.trimble.com/alloy-gnss/en-us/NMEA-0183messages_MessageOverview.html